File: src/main/java/space/mori/regenblock/Event/RegenBlockEventConfigSave.java
package space.mori.regenblock.Event;

import org.bukkit.event.*;

public class RegenBlockEventConfigSave extends Event
{
    private static final HandlerList handlers;
    
    static {
        handlers = new HandlerList();
    }
    
    public HandlerList getHandlers() {
        return RegenBlockEventConfigSave.handlers;
    }
    
    public static HandlerList getHandlerList() {
        return RegenBlockEventConfigSave.handlers;
    }
}

File: src/main/java/space/mori/regenblock/Event/RegenBlockEventConfigSave.kt
package space.mori.regenblock.event

import org.bukkit.event.*

class RegenBlockEventConfigSave : Event() {

    override fun getHandlers(): HandlerList {
        return RegenBlockEventConfigSave.handlerList
    }

    companion object {
        @JvmStatic val handlerList: HandlerList = HandlerList()

    }
}
